tests/shipping/models/Delivery - Boleto.model.js

```diff
@@ -54,7 +54,7 @@ export default function test(account) {
       cy.contains(email).should('be.visible')
       cy.contains('Fernando Coelho').should('be.visible')
       cy.contains('5521999999999').should('be.visible')
-      cy.contains('Boleto bancário').should('be.visible')
+      cy.contains('Boleto Bancário').should('be.visible')
```

Comment:
⛏ Is the case important here? Could use `{ matchCase: false }` or a regex `/boleto bancário/i`.

Reply:
Great tip! I'll do it on another PR.